src/guide/iroha-2.md

```diff
@@ -1,140 +1,84 @@
 # Iroha 2 vs. Iroha 1
 
-Iroha 2 is a **complete** re-write of
-[Hyperledger Iroha](https://iroha.readthedocs.io/en/develop/index.html) in
-Rust. As of writing, the two projects are developed concurrently.
+<!-- TODO: consider rewriting completely to accommodate the style change -->
 
-In this introduction we outline the differences between the two versions of
-Iroha and highlight the new features of Iroha 2. It should be of particular
-interest to those who are already familiar with Iroha but wish to upgrade
-and start using the newer version.
+Iroha 2 is a **complete** rewrite of [Hyperledger Iroha](https://iroha.readthedocs.io/en/develop/index.html) in Rust. As of writing, the two projects are developed concurrently.
```

Comment:
Iroha 1 isn't being actively developed anymore

Reply:
fixed in commit 68ae813491f1d7b0c5bdaa8350b44d17772179f8